styles/05-objects/badge.css

```diff
@@ -0,0 +1,45 @@
+.badge-wrapper{
+  position: relative;
+  width: fit-content;
+  margin: 3ch;
+  display: inline;
+}
+
+.badge{
+  align-items: center;
+  border-radius: 50%;
+  display: inline-flex;
+  font-weight: 600;
+  justify-content: center;
+  padding: 5px;
+  position: absolute;
+}
+.badge-right{
+  right: -8px;
+}
+.badge-left{
+  left:-4px;
+  transform: translate(0%, -20%)
+}
+button .badge-sm{
+  top:-5px;
+}
+button .badge-lg{
+  top:-15px;
+}
+.badge-sm{
```

Comment:
Do we need xs also to be in line with avatar sizes?

Reply:
will think about it

Reply:
Found  use case for xs avatars
<img width="98" alt="image" src="https://user-images.githubusercontent.com/82398518/153767409-837a1ca1-0823-4394-877d-838f4714c66d.png">